components/sysutils/rdiff-backup/history

```diff
@@ -1 +1 @@
-backup/rdiff-backup-26@1.3.3,5.11-2016.0.1.2
+backup/rdiff-backup-27@1.3.3-2020.0.1.2
```

Comment:
Two things here:
1. You'll need to keep the old entry and just have to add a new one for 2.7
2. You'll need to add a full FMRI with incremented COMPONENT_REVISION. Thus, the new entry should be
backup/rdiff-backup-27@1.3.3,5.11-2020.0.1.3